definitions/yok.d.ts

```diff
@@ -24,6 +26,8 @@ interface IInjector extends IDisposable {
 	isValidHierarchicalCommand(commandName: string, commandArguments: string[]): boolean;
 	getChildrenCommandsNames(commandName: string): string[];
 	buildHierarchicalCommand(parentCommandName: string, commandLineArguments: string[]): any;
+	publicApi: any;
+	_publicApi: any;
```

Comment:
Can we just use a private field in the publicApi itself? For example: `publicApi.__modules__`